### DISCUSSION TITLE
In-App Subscription for Store vendor in Fluxstore Manager App

### DISCUSSION THREAD

**Discussion by shubhamrane48** [inspireui/support#5239] [Working on Features]
Hello,

This is the best solution in manager app for monthly subscription to use full applications features.
Plugin link:
1. https://woocommerce.com/products/woocommerce-subscriptions/

**Comment by aliadnandeveloper** [comment]
This feature is very important, subscriptions must be added within the application

**Comment by Thedeep1** [comment]
This is the best solution in manager app for monthly subscription to use full applications features.

**Comment by revoru** [comment]
Imagine that you impose a monthly subscription on sellers, it will be profitable for you

**Comment by aliadnanidev** [comment]
This is the best solution in manager app for monthly subscription to use full applications features.

**Comment by thedoore** [comment]
This is the best solution

**Comment by idevgoone** [comment]
The best feature can be in the application

**Comment by tadamk** [comment]
This feature is very required this your solution. The only one until you earn from the app by charging sellers subscriptions

**Comment by Thedeep1** [comment]
Please support

**Comment by tadamk** [comment]
This feature is very important, subscriptions must be added within the application

**Comment by Thedeep1** [comment]
Ok

**Comment by NXW2019** [comment]
it is better to think about making subscribe to a subscription plan (a monthly flat rate) with the suppliers before publishing a product. and I think for DOKAN users this plugin will be better

plugin link: 
https://wedevs.com/dokan/modules/subscription/

**Reply by Thedeep1** [reply]
Wcfm ?

**Reply by NXW2019** [reply]
DOKAN

**Comment by NXW2019** [comment]
no dokan please please

**Reply by Thedeep1** [reply]
Ok

**Comment by aliadnandeveloper** [comment]
ok

**Comment by Thedeep1** [comment]
very important feature

**Reply by Thedeep1** [reply]
Yoo

**Comment by Thedeep1** [comment]
:/

**Comment by Uhtredtb** [comment]
Indeed, please make it happens @minhcasi

**Comment by Yan-Aleman** [comment]
Good idea 💡
